src/backend/looped/vehicle/fly.cpp

```diff
@@ -32,10 +32,34 @@ namespace big
 				last_vehicle = vehicle;
 			}
 
-			if (vehicle)
+			if (vehicle && CAM::GET_FOLLOW_PED_CAM_VIEW_MODE() != CameraMode::FIRST_PERSON)
 			{
-				Vector3 cam_pos = CAM::GET_GAMEPLAY_CAM_ROT(0);
-				ENTITY::SET_ENTITY_ROTATION(vehicle, cam_pos.x, cam_pos.y, cam_pos.z, 1, true);
+				Vector3 cam_rot = CAM::GET_GAMEPLAY_CAM_ROT(0);
+
+				/*
+				* Leaving this for experimentation in the future, but vehicle flying in first person is very unpleasant either way
```

Comment:
Does this really need to stay?

Reply:
I don't understand. Do you mean cam_rot or the comment or something else?
The whole thing isn't even close to being finished.

Reply:
I reviewed line 40. Also, if it is not done, convert this to a draft PR.

Reply:
@Rxann Nobody asked you.

Reply:
> @Rxann Nobody asked you.

When I run the formatter and it edits 3 lines of imports based on the defined formatting rules of YimMenu it is a problem, but when 29 lines of code that will not be used are left, it is not an issue?

Reply:
The virgin whitespace vs the chad comment.

His comments are relevant to potential developers of this feature in knowing what we have tried, versus what we haven't. 

You running the clang formatter on every file you touch is absolutly annoying, and there's no real reason to do it, as none of the other developers do it.

Reply:
It is not my fault other developers don't format the files they commit.

Reply:
It is your fault to check commits before you push them to the repo.

Reply:
> I reviewed line 40. Also, if it is not done, convert this to a draft PR.

Probably a fine idea moving forward.